WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -85,16 +110,36 @@ public boolean onOptionsItemSelected(final MenuItem item) {
         return super.onOptionsItemSelected(item);
     }
 
-    private void refreshUsersList(String dotComBlogId, final int localTableBlogId) {
-        PeopleUtils.fetchUsers(dotComBlogId, localTableBlogId, new PeopleUtils.FetchUsersCallback() {
+    private void fetchUsersList(String dotComBlogId, final int localTableBlogId, final int offset) {
+        if (mPeopleEndOfListReached) {
+            return;
+        }
+
+        FragmentManager fragmentManager = getFragmentManager();
+        final PeopleListFragment peopleListFragment = (PeopleListFragment) fragmentManager
+                .findFragmentByTag(KEY_PEOPLE_LIST_FRAGMENT);
+        if (peopleListFragment != null && offset > 0) {
```

Comment:
There's no loading indicator the first time the people list is shown, so user is left looking at an empty screen until the request completes. Maybe there should be a progress bar when the list is empty?